content/03-reference/01-tools-and-interfaces/06-prisma-studio.mdx

```diff
@@ -7,7 +7,13 @@ experimental: true
 
 ## Overview
 
-Prisma Studio is a visual editor for your database. To run Prisma Studio, run the following command in your terminal:
+Prisma Studio is a very powerful database GUI where you can interact with your data.
```

Comment:
let's try to be a bit more "neutral" here -> "very powerful database GUI" is a bit too much IMO 🙈